docs/guide/execute-transaction.md

```diff
@@ -212,4 +212,18 @@ const transaction: wtypes.TransactionAndSign = {
 }
 
 const res = await executeTransaction(deployer, transaction);
-```
\ No newline at end of file
+```
+
+## SignTransactions function
+
+This function takes array of `TransactionAndSign` objects and returns raw signed transaction
+
+```
+const transaction: wtypes.TransactionAndSign = [{
+  transaction: SDKTx,
+  sign: {sign: wtypes.SignType.SecretKey, fromAccount: bobAcc}
+}]
+const rawSign = signTransactions(transaction)
+```
+
+`rawSign` has array of raw signed transactions.
```

Comment:
Could you please add an example in a run script (maybe in `/asa`) ? And link it here?

Reply:
ok